tagging/apps/merge_tags.py

```diff
@@ -103,16 +101,10 @@ def merge_tags(
 ) -> None:
     LOGGER.info(f"Trying to merge tag: {merged_tag}")
 
-    # This allows to rerun the script without having to remove the manifest manually
-    try:
-        docker["manifest", "rm", merged_tag] & plumbum.FG
```

Comment:
This was wrong and it has to be done before every call to `docker manifest create`. Even if `docker manifest create` failed, it could have created local manifest, which we have to delete before next attempt.